server/lib/buff_server/packages.ex

```diff
@@ -50,7 +53,10 @@ defmodule BuffServer.Packages do
       {:error, %Ecto.Changeset{}}
 
   """
-  def create_package(attrs \\ %{}) do
+  def create_package(attrs \\ %{}, artifact_binary) do
```

Comment:
i would add this to the attrs.. if you require it you can do:
```suggestion
  def create_package(%{artifact_binary: artifact_binary} = attrs) do
```

Reply:
👍